dev/unix/install.sh.in

```diff
@@ -88,6 +88,37 @@ notion_create_tree() {
   mkdir -p "${INSTALL_DIR}"/tmp
 }
 
+# Safely handle an existing installation by removing it given user permission to
+# do so. If the user has no existing installation, this is a no-op.
+notion_check_existing_installation() {
+  local INSTALL_DIR="$(notion_install_dir)"
+  local NOTION_BIN="${INSTALL_DIR}/notion"
+
+  if [[ ! -z $INSTALL_DIR && -x $NOTION_BIN ]]; then
```

Comment:
Good practice is to quote test vars, and using `-n` is a little clearer than `! -z`, so something like:
```bash
if [[ -n "$INSTALL_DIR" && -x "$NOTION_BIN" ]]; then
```